src/Java.Interop.Tools.TypeNameMappings/Java.Interop.Tools.TypeNameMappings/JavaNativeTypeManager.cs

```diff
@@ -167,6 +167,10 @@ public static string ToJniName (Type type)
 				"java/lang/Object";
 		}
 
+		static readonly Lazy<Type> IJavaPeerableType = new Lazy<Type> (() =>
+			Type.GetType ("Java.Interop.IJavaPeerable, Java.Interop", throwOnError: true)!
+		);
```

Comment:
@jonpryor wait would this `throw` if called from an MSBuild task? We have those...